lib/processors/minifier.js

```diff
@@ -274,6 +274,7 @@ export default async function({
 			filename,
 			dbgFilename,
 			code,
+			resourcePath: resource.getPath(),
```

Comment:
`resourcePath` is already available as local variable. Or is the path changed in between?

Reply:
Thanks, you're right! Corrected 👍